src/pages/TrialMap.page

```diff
@@ -25,34 +28,53 @@
     <div id="map"/>
 
     <script>
-      var startCoordinates = {lat: {!trial.Start_point__Latitude__s}, lng: {!trial.Start_point__Longitude__s}};
-      var endCoordinates = {lat: {!trial.End_point__Latitude__s}, lng: {!trial.End_point__Longitude__s}};
+    function initMap() {
+
```

Comment:
Basically what I have done is to save the coordinates in a string and parse back to number and get a value of 0 for every Coordinate that is null. I know this may be a cheat and probably is very inefficient , but I couldn't think of a better solution.